

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
uummm, I'm pretty sure that you don't want whatever this is in the repo where everyone can see it ;-)